src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -13418,6 +13418,8 @@ public void AppendFormatted<T>(T value, int alignment) { }
         public void AppendFormatted<T>(T value, int alignment, string? format) { }
         public void AppendFormatted<T>(T value, string? format) { }
         public void AppendLiteral(string value) { }
+        public void Clear() { throw null; }
```

Comment:
Not: isn't the convention just `{ }` if there's no return/out?

Reply:
Yup.